elf/elf.h

```diff
@@ -1960,6 +1971,7 @@ struct RV64BE {
   static constexpr u32 plt_hdr_size = 32;
   static constexpr u32 plt_size = 16;
   static constexpr u32 pltgot_size = 16;
+  static constexpr u32 tlsdesc_words = 4;
```

Comment:
I wonder if it is still a good idea to inline four words into `.got` instead of reserving two words like for other psABIs. With this optimization, the RISC-V psABI's TLSDESC would diverge from others. My understanding is that, as a return, we can eliminate one indirection for each TLV reference for dlopen()'ed DSOs, which seems like a relatively rare use case. I'm not sure if this micro-optimization was a good trade-off. What do you think?

Reply:
I don't have a strong opinion either. For the glibc implementation there was no complications, I just had to assign the values directly instead of calling `_dl_make_tlsdesc_dynamic` which allocates and fills in the structure.

The inline design is in fact simpler to implement, since we don't need to worry about allocation or keeping track of the list of allocations, but this doesn't matter much for cross-arch libcs since they will have a common helper to do that anyway.

Perhaps we could ask on the spec PR what others think of this.